index.md

```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
+
+**Having a code review process will simulatenously upskill your developers and improve the quality of your code.**
+
+Get in touch today to learn about how code review can help your software development practises. This module can be customised to include reviewing someones code as part of a demonstration.
 
 ### Continuous Integration (CI)
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+Using Continuous Integration technologies like Gitlab Actions or CircleCI is a very cheap and easy way to increase the reliability of your software. In conjunction with source control and automated tests, every change to the code is checked to actually do what it is supposed to do. This frees developers up from testing changes by hand and to focus on their important work.
+
+This module will cover setting up a basic continuious integration system using Gitlab Actions. It will also cover what a good CI should be doing.
+
+**Continuous Integration allows developers to make changes confidently and efficently.**
+
+This module can be adjusted to use examples from your preferred technologies. After the course I'd be happy to help with troubleshooting any problems that arise.
+
 
 ## Pricing
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
\ No newline at end of file
+Each of the modules cost £1200 to deliver. An on-going consulting after the talks is £600 per day, or £100 per hour.
+
+I am deeply passionate about helping domain experts become better software engineers. If this pricing structure doesn't work for you, or you want to tailor the course to your business needs then please get in touch and we can discuss a pricing structure that works for you.
```

Comment:
I like this bit: "to tailor the course to your business needs then please get in touch and we can discuss a pricing structure that works for you."